test/lit/passes/legalize-and-prune-js-interface.wast

```diff
@@ -167,9 +169,17 @@
     ;; This will be pruned.
     (unreachable)
   )
+
+  ;; CHECK:      (func $export-exn (type $4) (result exnref)
+  ;; CHECK-NEXT:  (ref.null noexn)
+  ;; CHECK-NEXT: )
+  (func $export-exn (export "export-exn") (result exnref)
+    ;; This will be pruned.
```

Comment:
Where do we see that it is pruned? (The CHECK lines are the same, so..)

Reply:
The export is pruned, specifically. It is exported in the input but not in the output.